mscore/pagesettings.cpp

```diff
@@ -304,7 +304,13 @@ void PageSettings::applyToScore(Score* s)
       s->undoChangeStyleVal(Sid::pageOddBottomMargin, oddPageBottomMargin->value() * f);
       s->undoChangeStyleVal(Sid::pageOddLeftMargin, oddPageLeftMargin->value() * f);
       s->undoChangeStyleVal(Sid::pageTwosided, twosided->isChecked());
-      s->undoChangeStyleVal(Sid::spatium, spatiumEntry->value() * f1);
+
+      qreal oldSpatium = s->spatium();
+      qreal newSpatium = spatiumEntry->value() * f1;
+      if (oldSpatium != newSpatium)
+            s->spatiumChanged(oldSpatium, newSpatium);
```

Comment:
Shouldn't `spatiumChanged()` be called *after* actually changing it? Or does it not matter?

Reply:
You're right, it's indeed better to call fter changing it.